Automattic/woocommerce-services
# Replace/calypso form legend

## Description
Similar to https://github.com/Automattic/woocommerce-services/pull/2344, There is no comparable WordPress nor WooCommerce FormLegend. I moved the Calypso submodule's FormLegend over.

I also appended `wcs-client` to import so we know this has been migrated.

### Related issue(s)
Closes https://github.com/Automattic/woocommerce-services/issues/2591
Relates to https://github.com/Automattic/woocommerce-services/issues/2333



### Steps to test & screenshots/GIFs
1. Pull this branch, recompile asset with `npm run start`
2. Go to WooCommerce > Status > WooCommerce Shipping & Tax, http://localhost/wp-admin/admin.php?page=wc-status&tab=connect
3. Scroll to the bottom and these should show properly:
![image](https://user-images.githubusercontent.com/572862/199840238-ac29e7de-fe51-41c2-9b5f-c115000d3016.png)


### Checklist
